packages/entity-server/src/__tests__/__integration__/EntityDescendantRoutes.test.ts

```diff
@@ -0,0 +1,215 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { TestableEntityServer, setupTestApp, tearDownTestApp } from '../testUtilities';
+import { getEntitiesWithFields, sortedByCode, COUNTRIES } from './fixtures';
+
+describe('descendants', () => {
+  let app: TestableEntityServer;
+
+  beforeAll(async () => {
+    app = await setupTestApp();
+    app.grantAccessToCountries(COUNTRIES);
+  });
+
+  afterAll(() => tearDownTestApp(app));
+
+  describe('/hierarchy/<hierarchyName>/<entityCode>/descendants', () => {
```

Comment:
Existing tests (eg in `meditrak-server`) generally use express endpoint syntax in their descriptions

```suggestion
  describe('/hierarchy/:hierarchyName/:entityCode/descendants', () => {
```

I slightly prefer this syntax too, but that's a detail - happy either way!

Reply:
Yeah I like it too, switched 👍